roles/ceph-mon/defaults/main.yml

```diff
@@ -69,6 +69,7 @@ ceph_mon_docker_interface: eth0
 #ceph_mon_docker_subnet: # subnet of the ceph_mon_docker_interface
 ceph_mon_docker_username: ceph
 ceph_mon_docker_imagename: daemon
+ceph_osd_docker_tag: latest
```

Comment:
Why osd and not mon? I mean you're modifying the mon role so I was expecting to see this variable being named ceph_mon_docker_tag instead